lblod/frontend-gelinkt-notuleren
# Set-up basic creation-flow of installation meetings for municipalities

### Overview
This PR sets-up a basic creation-flow for installation meetings concerning municipal councils (gemeenteraden).
Specifically, it adds the following:
- Updates to the ember-data model definitions in order to support 'installatievergadering' resources.
- Modifications to the meeting inbox page in order to correctly show installation meetings and support the creation of installation meetings alongside common meetings.
- Support the creation of a basic installation meeting concerning a municipal council. At the moment, these meetings contain dummy hard-coded agendapoints.
- Some small adjustments to the meeting page when an installation-meeting is opened (mostly in terms of vocabulary used)

TODOS:
- [x] Internationalization
- [ ] (In the future) Move away from hard-coded meetings
- [ ] (In the future) Support different types of installation meetings.

##### connected issues and PRs:
[GN-4828](https://binnenland.atlassian.net/browse/GN-4828)
https://github.com/lblod/app-gelinkt-notuleren/pull/187


### Setup
This PR should be tested in combination with https://github.com/lblod/app-gelinkt-notuleren/pull/187

### How to test/reproduce
- Set-up the backend and start the frontend
- Log in as a municipality and visit the meeting inbox page
- Using the new meeting-creation menu, create a sample installation meeting
- Ensure that an installation meeting has been created with 9 (empty) sample agendapoints. 
- Notice that some terms have been adapted on the meeting-edit page to reflect you are editing an installation meeting
- On the meeting inbox page, you should be able to differentiate common meeting and installation meetings.

### Challenges/uncertainties




### Checks PR readiness
